bin/GameIndex.dbf

```diff
@@ -6505,6 +6505,10 @@ Serial = SLED-52488
 Name   = The Suffering [Demo]
 Region = PAL-E
 ---------------------------------------------
+Serial = SLED-52597
```

Comment:
Might be worth to check if the crc id is different, and if that needs to be added to crc database as well for hw renderer.

Can you tell me the crc id of the demo so I can check?

Reply:
![image](https://user-images.githubusercontent.com/17685135/71359201-87f2c800-2583-11ea-95a7-c0e442075213.png)


this is what pcsx2  says

Reply:
![image](https://user-images.githubusercontent.com/17685135/71359328-fb94d500-2583-11ea-90bd-c68688d06bc4.png)

Reply:
Thanks, it's already on the list.